src/postgres.h

```diff
@@ -6,6 +6,10 @@
 #include <producer.h>
 #include <stdlib.h>
 #include <libpq-fe.h>
+#include <pthread.h>
+#ifdef __linux__
+#include <sys/prctl.h>
```

Comment:
Nitpick:
`sys/prctl.h` is available on all the unix flavours we care about so `#ifdef __linux__` is in theory unnecessary. Personally I'm in favour of moving such compatibility bits into a file of its own and provide a wrapper function for the rest of the code to use.

Reply:
That's what autotools do for you btw.
We might define HAVE_SYS_PRCTL_H and use that macro.